mirdata/datasets/good_sounds.py

```diff
@@ -0,0 +1,290 @@
+
+
+"""
+import json
+import os
+from typing import Optional, Tuple, BinaryIO
+
+import librosa
+import numpy as np
+from mirdata import download_utils, jams_utils, core, io
+
+BIBTEX = """@inproceedings{romani2015real,
+  title={A Real-Time System for Measuring Sound Goodness in Instrumental Sounds},
+  author={Romani Picas, Oriol and Parra Rodriguez, Hector and Dabiri, Dara and Tokuda, Hiroshi and Hariya, Wataru and Oishi, Koji and Serra, Xavier},
+  booktitle={Audio Engineering Society Convention 138},
+  year={2015},
+  organization={Audio Engineering Society}
+}"""
+REMOTES = {
+    "packs": download_utils.RemoteFileMetadata(
+        filename="packs.json",
+        url="https://zenodo.org/record/4588740/files/packs.json?download=1",
+        checksum="3b512c280f8be64ccb59b0b294e84610",
+        destination_dir=".",
+    ),
+    "ratings": download_utils.RemoteFileMetadata(
+        filename="ratings.json",
+        url="https://zenodo.org/record/4588740/files/ratings.json?download=1",
+        checksum="b50b95fc7eb996b31a7cd290070f8059",
+        destination_dir=".",
+    ),
+    "sounds": download_utils.RemoteFileMetadata(
+        filename="sounds.json",
+        url="https://zenodo.org/record/4588740/files/sounds.json?download=1",
+        checksum="a60d90a964fd567ebc2e4b4e3f2990f2",
+        destination_dir=".",
+    ),
+    "takes": download_utils.RemoteFileMetadata(
+        filename="takes.json",
+        url="https://zenodo.org/record/4588740/files/takes.json?download=1",
+        checksum="318e840031397314907e7f9420f2abeb",
+        destination_dir=".",
+    ),
+    "all": download_utils.RemoteFileMetadata(
+        filename="good-sounds.zip",
+        url="https://zenodo.org/record/820937/files/good-sounds.zip?download=1",
+        checksum="2137bbb2d32c1d60aa51e1301225f541",
+        destination_dir=".",
+    ),
+}
+
+
+class Track(core.Track):
+    """GOOD-SOUNDS Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        track_id (str): track id of the track
+        audio_path (str): Path to the audio file
+
+    Cached Properties:
+        audio (tuple):
+        get_ratings_info (dict):  Some musicians rated some sounds in a 0-10 goodness scale for the user evaluation of the first
+            project prototype. Please read the paper for more detailed information.
+        get_pack_info (dict):
+        get_sound_info (dict): The entity containing the sounds annotations.
+        get_take_info (dict): A sound can have several takes as some of them were recorded using different microphones at the same
+            time. Each take has an associated audio file.
+    """
+
+    def __init__(
+        self,
+        track_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            track_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
```

Comment:
General comment - I think it might make sense to change the way you have structured the Track object. Instead of the four cached properties, it could instead have a lot of attributes/properties for each of the keys in the four current dictionaries. E.g. right now, all the most important information is within the dictionaries returned by the 4 annotation types. Instead, would it make sense to pull each of the dictionary key/values into Track properties? e.g. Track.instrument, Track.note, Track.octave, etc. What do you think?

Reply:
We have decided to add only the most usefull classes!

Reply:
added some usefule labels!

Reply:
done